2.0.0-beta.2/-modules.html

```diff
@@ -0,0 +1,143 @@
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1" charset="UTF-8">
+    <title>Modules</title>
+    <link href="images/logo-icon.svg" rel="icon" type="image/svg">
+    <script>var pathToRoot = "";</script>
+    <script type="text/javascript" src="scripts/sourceset_dependencies.js" async="async"></script>
+    <link href="styles/style.css" rel="Stylesheet">
+    <link href="styles/logo-styles.css" rel="Stylesheet">
+    <link href="styles/jetbrains-mono.css" rel="Stylesheet">
+    <link href="styles/main.css" rel="Stylesheet">
+    <script type="text/javascript" src="scripts/clipboard.js" async="async"></script>
+    <script type="text/javascript" src="scripts/navigation-loader.js" async="async"></script>
+    <script type="text/javascript" src="scripts/platform-content-handler.js" async="async"></script>
+    <script type="text/javascript" src="scripts/main.js" async="async"></script>
+  </head>
+  <body>
+    <div id="container">
+      <div id="leftColumn">
+        <div id="logo"></div>
+        <div id="sideMenu"></div>
+      </div>
+      <div id="main">
+        <div id="leftToggler"><span class="icon-toggler"></span></div>
+        <script type="text/javascript" src="scripts/pages.js"></script>
+        <script type="text/javascript" src="scripts/main.js"></script>
+        <div class="main-content" id="content" pageIds=".ext/allModules///PointingToDeclaration//0">
+          <div class="navigation-wrapper" id="navigation-wrapper">
+            <div class="breadcrumbs"></div>
+            <div class="pull-right d-flex">
+              <div id="searchBar"></div>
+            </div>
+          </div>
+          <div class="cover ">
+            <h2 class="">All modules:</h2>
+            <div class="table"><a data-name="%5B.ext%2Flibnavigation-android%2F%2F%2FPointingToDeclaration%2F%5D%2FMain%2F0" anchor-label="libnavigation-android" id="%5B.ext%2Flibnavigation-android%2F%2F%2FPointingToDeclaration%2F%5D%2FMain%2F0" data-filterable-set=""></a>
+              <div class="table-row">
+                <div class="main-subrow ">
+                  <div class="w-100"><span class="inline-flex"><a href="libnavigation-android/libnavigation-android/index.html">libnavigation-android</a><span class="anchor-wrapper"><span class="anchor-icon" pointing-to="%5B.ext%2Flibnavigation-android%2F%2F%2FPointingToDeclaration%2F%5D%2FMain%2F0"></span>
+                        <div class="copy-popup-wrapper "><span class="copy-popup-icon"></span><span>Link copied to clipboard</span></div>
+                      </span></span></div>
+                  <div><span class="brief-comment"></span></div>
+                </div>
+              </div>
```

Comment:
The page is gone, but it's still showing up in the table of contents. I think removing this should do the trick. 

```suggestion
```